addons/pos_stripe/static/src/js/payment_stripe.js

```diff
@@ -193,10 +194,31 @@ let PaymentStripe = PaymentInterface.extend({
         }
     },
 
+    getCardBrandFromPaymentMethodDetails(paymentMethodDetails) {
```

Comment:
you can simply call this `getCardBrand`
The short name is enough because in the code you will have `getCardBrand(xyz.paymentMethodDetails)` so it will be very obvious what's happening.